integration/login_logout_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/go-check/check"
+)
+
+func init() {
+	check.Suite(&LoginLogoutSuite{})
+}
+
+type LoginLogoutSuite struct {
+	regV2WithAuth *testRegistryV2
+}
+
+func (s *LoginLogoutSuite) SetUpSuite(c *check.C) {
+	_, err := exec.LookPath(skopeoBinary)
+	c.Assert(err, check.IsNil)
+}
+
+func (s *LoginLogoutSuite) TearDownSuite(c *check.C) {
+
+}
+
+func (s *LoginLogoutSuite) SetUpTest(c *check.C) {
+	s.regV2WithAuth = setupRegistryV2At(c, privateRegistryURL1, true, false)
```

Comment:
Why not create/tear down the registry in `SetUpSuite`/`TearDownSuite` instead? The registry carries no state AFAICS, and it would be faster.

Or, preferably, change `SkopeoSuite` that way, and make these tests parts of `SkopeoSuite`; that would get rid of even more overhead. Besides, `SkopeoSuite` already deals with authentication, so it would make sense to have everything together.

(I don’t care whether the new tests are in `login_logout_test.go` or in `check_test.go`.)